demos/notebooks/demo_pipeline.ipynb

```diff
@@ -610,10 +1023,736 @@
+    "\n",
+    "Viewing the residuals (what the model doesn't explain) can be extremely useful: if you end up seeing lots of neural activity in the residual movie, that means your model is leaving something important out, and you might need to go tweak some parameters. Indeed, in Caiman's online algorithms, this is how we decide whether to add new neurons after initialization. If there is something that smells neural in the residual buffer, then it's time to add a new neuron to `AC`! In the offline algorithms, it just means you might need to go check your parameters. Note there is rarely a perfect model, so there will typically be some residual that raises concerns: you will have to use your judgment about whether it is worth chasing after.\n",
+    "\n",
+    "> The `play_movie()` method has an option to include the background from the model (`bf`) or not (this is the `include_bck` Boolean parameter). If you set it to `False`, it will subtract `bf` from the original movie and will not add it to the reconstructed movie. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# in case you are working from loaded data, recover the raw movie\n",
+    "Yr, dims, num_frames = cm.load_memmap(cnmf_refit.mmap_file)\n",
+    "images = np.reshape(Yr.T, [num_frames] + list(dims), order='F')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# press q to quit\n",
+    "cnmf_refit.estimates.play_movie(images, \n",
+    "                                q_max=99.9, \n",
+    "                                gain_res=0.5,\n",
+    "                                magnification=2,\n",
+    "                                include_bck=True,\n",
+    "                                use_color=True,\n",
+    "                                thr=0); # set to 0.1 to see contours"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Keep in mind that if you ran `select_components()` to remove rejects, the residual will contain activity from those components that were rejected during the component evaluation step because their shape was awry or the SNR was low."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Where next?\n",
+    "The goal of Caiman is to help you extract high-quality signals from your data. Your next step, of doing actual analysis on these results, is where your scientific analysis and visualization creativity can really shine as you move forward. What are the statistical features of these signals? How do they relate to other environmental, molecular, behavioral, and neuronal features that you are studying? What kinds of visualization and analysis tools can you build around this infrastructure? If you find/publish things that help, please share them with us, as we would love to hear about them! "
```

Comment:
Didn't you define "the goal of caiman" above? 
"is where your .. creativity can really shine" what is this sentence trying to do? Can we avoid this kind of phrasing?

Reply:
Ok this is probably a bit much. :rose: :fist_raised: